specificatie/readme.md

```diff
@@ -1,3 +1,4 @@
 De OAS3-specificatie is te vinden in het bestand openapi.yaml. In deze specificatie hebben we json-Hal toegepast.
 
-In de bestanden openapi.json en openapi_draft4.json zijn twee (functioneel gelijke) varianten van het json-schema te vinden. Dit kan toegepast worden bij bestandsuitwisseling. In deze bestanden komen de hal-specifieke componenten uit openapi.yaml niet voor. 
+In de bestanden openapi.json en openapi_draft4.json zijn twee (functioneel gelijke) varianten van het json-schema te vinden. Dit kan toegepast worden bij bestandsuitwisseling. In deze bestanden komen de hal-specifieke componenten uit openapi.yaml niet voor.
+Bij bestandsuitwisseling wordt het RD/Amersfoort(EPSG:28992) coördinatenstelsel gebruikt. 
```

Comment:
Je schrijft "Bij bestandsuitwisseling wordt het RD/Amersfoort(EPSG:28992) coördinatenstelsel gebruikt." Alleen bij bestandsuitwisseling, niet bij gebruik van de API. Of moet er iets bij dat het gebruikte coordinatenstelsel bij het gebruik van de API in de header wordt gezet?

Reply:
Klopt, in de API kan dat m.b.v. de header parameter Accept-Crs en wordt dat teruggekoppeld met de Content-Crs. Ik zal de specificatie aanpassen zodat voor deze twee headers verwezen wordt naar versie 1.3.0 van Haal-Centraal-common. In die versie is dat standaard geregeld. (RD / EPSG28992) is de enige mogelijke waarde en de default.